napari/_qt/qt_event_loop.py

```diff
@@ -142,6 +153,11 @@ def get_app(
             show_console_notification
         )
 
+    app.setWindowIcon(QIcon(kwargs.get('icon')))
+    if ipy_interactive is None:
+        ipy_interactive = SETTINGS.application.ipy_interactive
+    _try_enable_ipython_gui(ipy_interactive and 'qt')
```

Comment:
What does passing `False` or `None` to this do?

Reply:
it'll stop the ipython `active_eventloop`:

```python
In [1]: shell = get_ipython()

In [2]: shell.active_eventloop

In [3]: shell.enable_gui("qt")

In [4]: shell.active_eventloop
Out[4]: 'qt'

In [5]: shell.enable_gui(None)

In [6]: shell.active_eventloop

In [7]: from qtpy.QtWidgets import QWidget

In [8]: w = QWidget()

In [9]: w.show()  # doesn't show

In [10]: shell.enable_gui('qt')  # shows now

```

Reply:
@tlambert03 it's cute but I don't find it very readable. You might recall our snafu with the pytest skip markers. 😂 So I would suggest here:

```suggestion
    if ipy_interactive:
        _try_enable_ipython_gui('qt')
```

Reply:
that's not quite right either, because we need it to be `None` if `ipy_interactive` is `None` and `qt` otherwise.  I can change to a different syntax though.  (the pytest snafu was also a pytest issue IIRC ... not a `X and Y` issue)

Reply:
well, I learned something about IPython from this conversation!